addons/web_tour/static/src/js/tip.js

```diff
@@ -281,6 +356,16 @@ var Tip = Widget.extend({
 
         this.tip_opened = false;
 
+        if (this.$el.parent()[0] !== this.$ideal_location) {
+            // OSU TODO : modify to be cleaner, and check if detach is useful to do or not (don't think so)
+            // this.$el.detach();
+            if (this.$ideal_location.is(this.$anchor)) {
+                this.appendTo(this.$furtherIdealLocation);
+            } else {
+                this.appendTo(this.$ideal_location);
+            }
+        }
```

Comment:
I changed a bit the code in `attach_to` and there, so I think it shouldn't use the same code. `attach_to`'s scroll handler do an `appendTo()` on `this`. And know, in the `_build_bubble_mode` method, the `appendTo` function is on `this.$el`.
Because in the first method, we move our tip to a position from another one, so I need to go back in `_reposition()`.
But the second method just need to close the bubble. But, only closing the bubble made it move elsewhere. So, like in when we open the bubble, we need to replace correctly the tip, but not by going in `_reposition()` but by using the last position saved by `_reposition()`. 
I'll push the changes ;)